lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1347,6 +1347,12 @@ enum DebuggerBroadcastBit {
   eBroadcastBitProgressCategory = (1 << 3),
 };
 
+enum Severity {
+  eSeverityError,
+  eSeverityWarning,
+  eSeverityInfo,
```

Comment:
Can you comment that "Clang also calls these Remarks, it's conceptually the same level"

Reply:
I considered the latter, but we use `eSeverityRemark` so infrequently and not even always for `DiagnosticsEngine::Level::Remark` that I decided against it. The comment however seems like a good compromise.